### PR TITLE
Completed css-brochure-form exercise. -Janice Lee

### PR DESCRIPTION
**Pull request by janiicelee**
Screen Shot 2022-01-26 at 8 42 08 PM

### CODE REVIEW

**Review by codingcodymiller — CHANGES_REQUESTED** [review]
Hey Janice! The color of your header and submit button do not match the example, please revise and submit a new screenshot with your revision.

**Comment by janiicelee** [comment]
Screen Shot 2022-01-27 at 10 48 16 AM

**Review by codingcodymiller — CHANGES_REQUESTED** [review]
Hey Janice! Your `select` element seems to be slightly misaligned with the other `input` elements, and there is too much space between the `select` element and the submit `button`, please revise and submit a new screenshot with your revision.

**Comment by janiicelee** [comment]
Screen Shot 2022-01-28 at 11 31 54 AM

I think I tried my best on the submit button, but not sure if its accurate.. haha.

File: css-brochure-form/styles.css [hunk 1]
```diff
@@ -0,0 +1,50 @@
+form {
+  width: 50%;
+  margin: auto;
+  font-family: "Brandon Grotesque", sans-serif !important;
```
Comment (codingcodymiller) [inline css-brochure-form/styles.css:4]: 🤢 

Why `!important`? You can remove that.

**Review by codingcodymiller — CHANGES_REQUESTED** [review]

File: css-brochure-form/styles.css (hunk 1 above)
Comment (janiicelee) [reply css-brochure-form/styles.css:4]: I get this `font-family-no-missing-generic-family-keyword` error and I found the solution on google to fix that stylelint error..!

**Review by codingcodymiller — APPROVED** [review]

**Pull request merged**